index.html

```diff
@@ -45,6 +47,33 @@
         </nav>
   
         <main id="main-doc">
+            <section class="main-section" id="buiGuide">
+                <header><i class="bi bi-bookmark-fill"></i>Installation Guide</header>
+                <p><strong>A Simple HTML & CSS component library</strong></p>
+                <p>BoilerUI is the simple HTML & CSS component library with diffrent component that you can use for your next project. BoilerUI gives the base component to start your next project quickly, without doing everyting from scratch. We tried to keep thing simple as name says BoilerUI, so you can add more customization to it, according you your project need.</p>
+
+                <p>You can simply download or clone the github repo. File structure are given below</p>
+
+                <div class="bui-code-example">
+                <pre><code class="language-html">
+                /CSS/boilerui.css
```

Comment:
Indent it properly. If you want to remove the extra blank space you can use either white-space: pre/pre-wrap property or can give a negative left margin.